packages/eslint-config-angular/i18n-attributes-ignore.js

```diff
@@ -0,0 +1,50 @@
+module.exports = {
+    nimble: [
+        'action-menu-slot',
+        'appearance',
+        'appearance-variant',
+        'column-id',
+        'field-name',
+        'format',
+        'icon',
+        'id-field-name',
+        'key',
+        'key-type',
+        'orientation',
+        'severity',
+        'slot',
+        'theme'
+    ],
+    'systemlink-lib-angular': [
```

Comment:
What about just `systemlink` since these are really domains more than exact packages.

```suggestion
    systemlink: [
```

I'd also be up for exact packages. `@ni/nimble-angular`, `@ni-kismet/systemlink-lib-angular`, etc

Reply:
Change to just `systemlink`